bin/commands/init/stc/index.sh

```diff
@@ -238,14 +254,14 @@ else
   print_message
 fi
 
-if [ "${stc_existence}" = "true" ] &&  [ "${ZWE_CLI_PARAMETER_ALLOW_OVERWRITE}" != "true" ]; then
-  print_message "Skipped writing to ${proclib}(${mb}). To write, you must use --allow-overwrite."
+if [ "${any_stc_existence}" = "true" ] && [ "${ZWE_CLI_PARAMETER_ALLOW_OVERWRITE}" != "true" ]; then
```

Comment:
Compared to changes in v3, is `any_jcl_existence` missing in this check?

https://github.com/zowe/zowe-install-packaging/pull/4476/files#diff-b484fba27e1a85a0c0ee4165c1176eee21ae5b15a9fc68df4ceb64ed9551075fR289